int32 arm_control_mode
float64[5] joints
geometry_msgs/Pose end_effector_pose
geometry_msgs/Pose goal_pose
